examples~/quickstart/client/module_bindings/_Globals/SpacetimeDBClient.cs

```diff
@@ -158,5 +158,7 @@ protected override bool Dispatch(IEventContext context, Reducer reducer) {
 				_ => throw new ArgumentOutOfRangeException("Reducer", $"Unknown reducer {reducer}")
 			};
 		}
+
+        public SubscriptionBuilder<EventContext> SubscriptionBuilder() => new(this);
```

Comment:
I've done this manually for now, it will need codegen changes once the client codegen part of https://github.com/clockworklabs/SpacetimeDB/pull/1707 lands.